### DISCUSSION TITLE
Adjust screenshot resolution via script

### DISCUSSION THREAD

**Discussion by WitAmidstFolly** [Stellarium/stellarium#3208] [Q&A]
Is it possible to set the screenshot heigh/width in a script? Able to do this in the app in the tools settings, but the screenshot preference isn't saved when the programs closed and I can't as yet see a way to set it in a script, just wondering if it's possible

**Comment by 10110111** [comment]
Although it totally makes sense to expose this in the API, it seems choosing a custom size is not implemented.

**Comment by gzotti** [comment]
If you store your preferences, these will be stored as well. 
Yes, setting size (and other internals of StelMainView) is indeed not exposed to scripting.
